libs/dev_only_scripts/configToDb.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+require_once '/var/www/html/libs/php-commons/Db.php';
+
+$dir = '/var/www/html';
+
+checkTemplate($dir);
+
+function checkTemplate($folder) {
+    //if (is_dir($folder . '/.svn')) {
+    if (is_dir($folder . '/sources')) {
+        if (file_exists($folder . '/sources/DbConfig.php')) {
+            // we are in a portal
+            $portal = str_replace('/var/www/html', '', $folder);
+
+            // instantiate a national_leaf_launchpad db
+            $launchpad_db = new \Leaf\Db(getenv('DATABASE_HOST'), getenv('DATABASE_USERNAME'), getenv('DATABASE_PASSWORD'), 'national_leaf_launchpad');
+
+            // get the db name for this portal
+            $vars = array(':site_path' => $portal);
+            $sql = 'SELECT portal_database
+                    FROM sites
+                    WHERE site_path= BINARY :site_path';
+
+            $site_paths = $launchpad_db->prepared_query($sql, $vars)[0];
+
+            // instantiate a portal db
+            $db =  new \Leaf\Db(getenv('DATABASE_HOST'), getenv('DATABASE_USERNAME'), getenv('DATABASE_PASSWORD'), $site_paths['portal_database']);
+
+            // instantiate a Config.php
+            require_once $folder . '/sources/Config.php';
+            $config = new Portal\Config();
+
+            // populate db with config data
+            $sql = 'INSERT INTO `settings` (`setting`, `data`)
+                    VALUES (:setting, :data)
+                    ON DUPLICATE KEY UPDATE `setting`=:setting';
+
+            $vars = array(':setting' => 'heading',
+                    ':data' => $config->title);
+            $db->prepared_query($sql, $vars);
+
+            $vars = array(':setting' => 'subHeading',
+                    ':data' => $config->city);
+            $db->prepared_query($sql, $vars);
+
+            $data = json_encode($config->adPath, JSON_FORCE_OBJECT);
+            error_log(print_r($data, true));
+            $vars = array(':setting' => 'adPath',
+                    ':data' => json_encode($config->adPath, JSON_FORCE_OBJECT));
+            $db->prepared_query($sql, $vars);
+
+            $vars = array(':setting' => 'orgchartImportTags',
+                    ':data' => json_encode(Portal\Config::$orgchartImportTags, JSON_FORCE_OBJECT));
+            $db->prepared_query($sql, $vars);
+
+            $vars = array(':setting' => 'descriptionID',
+                    ':data' => $config->descriptionID);
+            $db->prepared_query($sql, $vars);
+
+            $vars = array(':setting' => 'emailPrefix',
+                    ':data' => Portal\Config::$emailPrefix);
+            $db->prepared_query($sql, $vars);
+
+            $vars = array(':setting' => 'emailCC',

```

Comment:
For emailCC and emalBCC do these even get used in our codebase from this config I don't believe this is necessary since theres not a setting for these on the Site Settings page as well. And they are just empty array's on all the sites configs.

Reply:
Here is how it's being used currently.

if (isset(Config::$emailCC) && count(Config::$emailCC) > 0) {
            foreach (Config::$emailCC as $recipient) {
                $this->addCcBcc($recipient);
            }
        }

Reply:
Guess this was meant to be added to settings on a greater level than email templates at some point lets just add it for now. They most likely are all empty arrays.